doc/src/modules/numeric-computation.rst

```diff
@@ -73,6 +73,18 @@ If you have array-based data this can confer a considerable speedup, on the
 order of 10 nano-seconds per element. Unfortunately numpy incurs some start-up
 time and introduces an overhead of a few microseconds.
 
+CuPy is a NumPy-compatible array library that mainly runs on CUDA, but has 
+increasing support for other GPUs manufacturer. It can in many cases be used as 
+a drop-in replacement for numpy. It is particu
+
+    >>> f = lambdify(x, expr, "cupy")
```

Comment:
You need to update the blacklisting of this file in sympy/utilities/runtests.py so that the doctests for it aren't run if cupy isn't installed.

Reply:
I'm happy to do this, but I think I need some help. Based on the contents of /runtests.py, I have no idea how to proceed with blacklisting the rst file.

Reply:
Ah, you meant sympy/testing/runtests.py, not sympy/utilities/runtests.py.

Reply:
Oh sorry. The file used to live in utilities for a long time, and old habits die hard.